server/utils.js

```diff
@@ -0,0 +1,29 @@
+//Probably should move more stuff into here
+
+var shortid = require("shortid")
+var logger = require("./logger")
+
+var slugify = function(str) {
```

Comment:
Perhaps something like this would work:

```js
function slugify(str) {
    return str.replace(/[^A-Za-z0-9]/g, " ").trim().replace(/\s{2,}/g, "-");
}
```